src/container/srv_container.c

```diff
@@ -5136,20 +5157,23 @@ static int
 cont_op_with_svc(struct ds_pool_hdl *pool_hdl, struct cont_svc *svc,
 		 crt_rpc_t *rpc, int cont_proto_ver)
 {
-	struct cont_op_in		*in = crt_req_get(rpc);
-	struct cont_open_bylabel_in	*olbl_in = NULL;
-	struct cont_open_bylabel_out	*olbl_out = NULL;
-	struct cont_destroy_bylabel_in	*dlbl_in = NULL;
-	struct rdb_tx			 tx;
-	crt_opcode_t			 opc = opc_get(rpc->cr_opc);
-	struct cont			*cont = NULL;
-	struct cont_pool_metrics	*metrics;
-	bool				 update_mtime = false;
-	int				 rc;
+	struct cont_op_in              *in       = crt_req_get(rpc);
+	struct cont_open_bylabel_in    *olbl_in  = NULL;
+	struct cont_open_bylabel_out   *olbl_out = NULL;
+	struct cont_destroy_bylabel_in *dlbl_in  = NULL;
+	struct rdb_tx                   tx;
+	crt_opcode_t                    opc  = opc_get(rpc->cr_opc);
+	struct cont                    *cont = NULL;
+	struct cont_pool_metrics       *metrics;
+	bool                            update_mtime = false;
+	bool                            dup_op       = false;
+	const char                     *clbl         = NULL;
+	char                            cuuid[37];
```

Comment:
[Not requesting changes] There's `DAOS_UUID_STR_SIZE`.

Reply:
Sure, will fix that up in a follow-on patch for this work, thanks.